src/Tests/Microsoft.NET.Sdk.BlazorWebAssembly.Tests/StaticWebAssetsBaselines/Publish60Hosted_Works.Publish.staticwebassets.json

```diff
@@ -1270,21 +1270,21 @@
       "OriginalItemSpec": "${ProjectRoot}\\Client\\bin\\Debug\\net6.0\\wwwroot\\_framework\\blazor.webassembly.js"
     },
     {
-      "Identity": "${ProjectRoot}\\Client\\obj\\Debug\\net6.0\\compress\\[[_framework/dotnet.6.0.0-rc.2.21452.2.[[hash]].js.br]]",
+      "Identity": "${ProjectRoot}\\Client\\obj\\Debug\\net6.0\\compress\\[[_framework/dotnet.6.0.2.[[hash]].js.br]]",
       "SourceId": "BlazorWasmHosted60.Client",
       "SourceType": "Project",
       "ContentRoot": "${ProjectRoot}\\Client\\bin\\Debug\\net6.0\\publish\\wwwroot\\",
       "BasePath": "/",
-      "RelativePath": "_framework/dotnet.6.0.0-rc.2.21452.2.[[hash]].js.br",
+      "RelativePath": "_framework/dotnet.6.0.2.[[hash]].js.br",
       "AssetKind": "Publish",
       "AssetMode": "All",
       "AssetRole": "Alternative",
-      "RelatedAsset": "${ProjectRoot}\\Client\\bin\\Debug\\net6.0\\wwwroot\\_framework\\dotnet.6.0.0-rc.2.21452.2.[[hash]].js",
+      "RelatedAsset": "${ProjectRoot}\\Client\\bin\\Debug\\net6.0\\wwwroot\\_framework\\dotnet.6.0.2.[[hash]].js",
       "AssetTraitName": "Content-Encoding",
       "AssetTraitValue": "br",
       "CopyToOutputDirectory": "Never",
       "CopyToPublishDirectory": "PreserveNewest",
-      "OriginalItemSpec": "${ProjectRoot}\\Client\\bin\\Debug\\net6.0\\wwwroot\\_framework\\dotnet.6.0.0-rc.2.21452.2.[[hash]].js"
+      "OriginalItemSpec": "${ProjectRoot}\\Client\\bin\\Debug\\net6.0\\wwwroot\\_framework\\dotnet.6.0.2.[[hash]].js"
```

Comment:
Why do we have a version here at all (I thought we replaced versions with `[[runtimeVersion]]` similar to `[[hash]]`. 

Is this something we'll have to keep up to date as the branding changes?

Reply:
Not sure why this is happening, but let's unblock the build for now, and we can take a look later.